types/msg.go

```diff
@@ -47,6 +47,7 @@ func (e Events) MarshalJSON() ([]byte, error) {
 func (e *Events) UnmarshalJSON(data []byte) error {
 	// make sure we deserialize [] back to null
 	if string(data) == "[]" || string(data) == "null" {
+		*e = nil
```

Comment:
This is just a question. What is the reason for setting nil here?

Reply:
Without this, when e has some events and this reads "[]", e does not be changed. Maybe this is not as expected. For example, https://github.com/line/wasmvm/pull/88/commits/0a2f4e2e5cc85a28508e3a338be17e9fe59366a3#diff-41f47570e26c8c3ff485fb26d89e5fb3c5468799c6b07fe3449dd2415730fd42R1197 does not change `events`'s value.